digitalocean/database/resource_database_kafka_topic.go

```diff
@@ -0,0 +1,573 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/digitalocean/godo"
+	"github.com/digitalocean/terraform-provider-digitalocean/digitalocean/config"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceDigitalOceanDatabaseKafkaTopic() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDigitalOceanDatabaseKafkaTopicCreate,
+		ReadContext:   resourceDigitalOceanDatabaseKafkaTopicRead,
+		UpdateContext: resourceDigitalOceanDatabaseKafkaTopicUpdate,
+		DeleteContext: resourceDigitalOceanDatabaseKafkaTopicDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDigitalOceanDatabaseUserImport,
```

Comment:
Looks like this needs an import function.  The database user import is a good example for a resource like this where it needs more than one identifier to import (ie the parent cluster's ID and the name).

https://github.com/digitalocean/terraform-provider-digitalocean/blob/fb899442bee65259dcc3810f5cde963dd4c79009/digitalocean/database/resource_database_user.go#L190

Reply:
Fixed